e2e-tests/upgrade/run

```diff
@@ -127,6 +127,7 @@ function prepare_cr_yaml() {
 		| yq w - "spec.backup.storages.minio.s3.region" "us-east-1" \
 		| yq w - "spec.backup.storages.minio.s3.bucket" "operator-testing" \
 		| yq w - "spec.backup.storages.minio.s3.endpointUrl" "http://minio-service:9000/" \
+		| yq w - "spec.backup.storages.minio.s3.insecureSkipTLSVerify" "true" \
```

Comment:
why ? do we have any issues?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/935/commits/578f6da7352dd65039c41db6e535f5eedc573afc